src/Samples/DownloadFile/DownloadFile.targets

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="DownloadLicense"
+          AfterTargets="PrepareForBuild">
+
+    <!-- Download a file -->
+    <DownloadFile
+      SourceUrl="https://raw.githubusercontent.com/Microsoft/msbuild/master/LICENSE"
+      DestinationFolder="$(MSBuildProjectDirectory)">
```

Comment:
Can you specify a path or something so this can be named `DownloadedFile` instead of `LICENSE`? I found the `.gitignore` confusing.